### ISSUE TITLE
connect forest Admin to our db - not finished

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/loopback-tutor-intern-8#30]

**Comment by chauhannishith** [comment]
Do you mean forestadmin.com?

**Comment by atherdon** [comment]
yep. i also have some code samples here:https://github.com/ChickenKyiv/recipe-dash you can fork it or as you familiar with loopback - check how to include code into server side.

**Comment by atherdon** [comment]
yep, this is where we were stuck at that repo. but maybe now we'll be able to move forward

**Comment by atherdon** [comment]
it also will be cool things related to payments stuff(#27) because Forest can connect to stripe too and display some admin like information like transactions list, etc...

**Comment by chauhannishith** [comment]
Ok i will try myself to get it working
and yeah i saw that we can push refunds and make address change even after an order has been submitted

**Comment by atherdon** [comment]
i didn't explore the latest versions. but i'm some sort of fun of their project. we also have a chat in december-january - the we're like 2 years strugling without cash and without rolling out a lot of features and  in the end of last year they got 12mil of investments and start to rising like a star!

**Comment by chauhannishith** [comment]
[https://github.com/ForestAdmin/forest-loopback/issues/7](url) Couldn't find anything related to loopback except this.
Does this even support loopback?

**Comment by atherdon** [comment]
it works with express and mongo. and should work with loopback too
